modules/conductor/src/main/java/org/opencastproject/event/handler/AssetManagerUpdatedEventHandler.java

```diff
@@ -242,10 +242,8 @@ public void handleEvent(final SeriesItem seriesItem) {
           logger.error("Error updating mediapackage {}", mp.getIdentifier().compact(), e);
         }
       }
-    } catch (NotFoundException e) {
-      logger.warn(e.getMessage());
-    } catch (IOException e) {
-      logger.warn(e.getMessage());
+    } catch (IOException | NotFoundException e) {
+      logger.warn("Unable to handle update event for series {}: {}", seriesItem, e.getMessage());
```

Comment:
Is it Ok, privacy wise, to also add the user "preUser"'s username? Since you have it handy already?

Reply:
Added